packaging/rpm/alvr.spec

```diff
@@ -1,35 +1,52 @@
 Name: alvr
-Version: 16.0.0-rc1
-Release: 1.0.0
+Version: 16.0.0
+Release: 0.0.1rc1
 Summary: Stream VR games from your PC to your headset via Wi-Fi
 License: MIT
-Source: https://github.com/alvr-org/ALVR/archive/refs/tags/v%{version}.tar.gz
+Source: https://github.com/alvr-org/ALVR/archive/refs/tags/v16.0.0-rc1.tar.gz
+URL: https://github.com/alvr-org/ALVR/
 ExclusiveArch: x86_64
-BuildRequires: alsa-lib-devel cairo-gobject-devel cargo clang-devel ffmpeg-devel gcc gcc-c++ ImageMagick libunwind-devel rust rust-atk-sys-devel rust-cairo-sys-rs-devel rust-gdk-sys-devel rust-glib-sys-devel rust-pango-sys-devel vulkan-headers vulkan-loader-devel
+BuildRequires: alsa-lib-devel cairo-gobject-devel cargo clang-devel ffmpeg-devel gcc gcc-c++ ImageMagick libunwind-devel openssl-devel rpmdevtools rust rust-atk-sys-devel rust-cairo-sys-rs-devel rust-gdk-sys-devel rust-glib-sys-devel rust-pango-sys-devel selinux-policy-devel vulkan-headers vulkan-loader-devel
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root
 Requires: ffmpeg steam
 Requires(post): policycoreutils
 Requires(postun): policycoreutils
+%define alvrBuildDir build/%{name}_server_linux
+# find-debuginfo.sh doesn't appear to be working
 %global debug_package %{nil} 
 
 %description
-ALVR is an open source remote VR display which allows playing SteamVR games on a standalone headset such as Gear VR or Oculus Go/Quest.
-
-%pre
-%define alvrBuildDir build/%{name}_server_linux
+ALVR is an open source remote VR display which allows playing SteamVR games on
+ a standalone headset such as Gear VR or Oculus Go/Quest.
 
 %prep
 %autosetup -D -n %{_builddir}
 
 %build
+# Set CXXFLAGS for ffmpeg
+export CXXFLAGS='-I/usr/include/ffmpeg'
 # Build ALVR
 cargo xtask build-server --release
 # Build SELinux policy
-rm -f packaging/selinux/alvr.pp.bz2
-make -f /usr/share/selinux/devel/Makefile -C 'packaging/selinux'
-bzip2 "packaging/selinux/%{name}.pp"
+rm -f 'packaging/selinux/%{name}.pp.bz2'
+make -f '/usr/share/selinux/devel/Makefile' -C 'packaging/selinux'
+bzip2 'packaging/selinux/%{name}.pp'
 
 %changelog
+* Tue Jul 27 2021 Trae Santiago <trae32566@gmail.com> - 16.0.0-0.0.1rc1
+    - Added rpmdevtools to BuildRequires so yum-builddeps adds it
+    - Added missing openssl-devel and selinux-policy-devel
+    - Broke description into multiple lines
+    - Added URL
+    - Updated Source to be static
+    - Added license as doc
+* Wed Jul 21 2021 Trae Santiago <trae32566@gmail.com> - 15.2.1-1.1.1
+    - Added CXXFLAGS
+* Tue Jul 20 2021 Trae Santiago <trae32566@gmail.com> - 15.2.1-1.1.0
+    - Changed source location for firewalld config
+    - Replaced all double quotes with single where possible
+    - Replaced all instances of alvr with name
+    - Made directory creation more readable
```

Comment:
I think you can purge the changelog since all these changes have been made before this was merged.